iron-collapse.html

```diff
@@ -278,19 +283,13 @@
       this.toggleClass('iron-collapse-opened', this.opened);
       this._updateTransition(false);
       this.notifyResize();
+      this._setTransitioning(false);
     },
 
-    /**
-     * Simplistic heuristic to detect if element has a parent with display: none
-     *
-     * @private
-     */
-    get _isDisplayed() {
```

Comment:
I feel dubious about this "feature" to detect if the element is displayed, but: are we sure it is safe to remove this check?